releases.json

```diff
@@ -3686,4 +3694,4 @@
       "1.3.3-1"
     ]
   }
-}
+}
```

Comment:
uhhh

Reply:
I don't know why it did that, maybe because it's second to last so maybe it's some problem with trailing comma that makes diff freak out

Reply:
No, you removed the terminating newline.

Reply:
Was that bad?

Reply:
Yes.